eo-maven-plugin/src/test/java/org/eolang/maven/WalkTest.java

```diff
@@ -23,14 +23,21 @@
 final class WalkTest {
 
     @Test
-    void findsFiles(@Mktmp final Path temp) throws Exception {
-        new HmBase(temp).save("", Paths.get("foo/hello/0.1/EObar/x.bin"));
-        new HmBase(temp).save("", Paths.get("EOxxx/bar"));
+    void findsFilesMatchingGlobPattern(@Mktmp final Path temp) throws Exception {
+        final String nonmatch = "foo/hello/0.1/EObar/x.bin";
+        final String match = "EOxxx/bar";
+        final String pattern = "EO**/*";
+        final int count = 1;
+        new HmBase(temp).save("", Paths.get(nonmatch));
```

Comment:
@a1kuat please check [this](https://github.com/objectionary/eo/issues/3997) ticket. Let's replace these `HmBase` occurrences with `Saved`